packages/anon-aadhaar-react/src/extractWitness.ts

```diff
@@ -0,0 +1,81 @@
+import { extractCert, extractSignature } from 'anon-aadhaar-pcd'
```

Comment:
I think this function should move to pcd package.

Reply:
Done!